lighthouse-core/audits/dobetterweb/no-vulnerable-libraries.js

```diff
@@ -16,6 +16,33 @@ const Audit = require('../audit.js');
 const Sentry = require('../../lib/sentry.js');
 const semver = require('semver');
 const snykDatabase = require('../../../third-party/snyk/snapshot.json');
+const i18n = require('../../lib/i18n/i18n.js');
+
+const UIStrings = {
+  /** Title of a Lighthouse audit that provides detail on Javascript libraries the page uses. This descriptive title is shown to users when all Javascript libraries are free of known security vulnerabilities. */
+  title: 'Avoids front-end JavaScript libraries' +
+    ' with known security vulnerabilities',
+  /** Title of a Lighthouse audit that provides detail on Javascript libraries the page uses. This descriptive title is shown to users when some detected Javascript libraries have known security vulnerabilities. */
+  failureTitle: 'Includes front-end JavaScript libraries' +
+    ' with known security vulnerabilities',
+  /** Description of a Lighthouse audit that tells the user why they should be concerned about the third party Javascript libraries that they use. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
+  description: 'Some third-party scripts may contain known security vulnerabilities ' +
+    'that are easily identified and exploited by attackers. ' +
+    '[Learn more](https://developers.google.com/web/tools/lighthouse/audits/vulnerabilities).',
+  /** [ICU Syntax] Label for the audit identifying the number of vulnerable Javascript libraries found. */
+  displayValue: `{itemCount, plural,
+    =1 {1 vulnerability detected}
+    other {# vulnerabilities detected}
+    }`,
+  /** Table column header for the version of the Javascript library found. */
+  columnVersion: 'Library Version',
+  /** Table column header for the count of vulnerabilities found within a JavaSscript library. */
+  columnVuln: 'Vulnerability Count',
+  /** Table column header for the severity of the vulnerabilities found within a Javascript library. */
```

Comment:
```suggestion
  /** Label for a column in a data table; entries will be the severities of the JavaScript-library vulnerabilities found.  */
```

Reply:
(where do the severity entry strings come from and should we be mapping them to translated strings in this file?)

Reply:
They come from `third-party/snyk/snapshot.json` and I will definitely i18n them, good catch.